scheduling/tasks.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"fmt"
 	"time"
+
+	"github.com/featureform/fferr"
```

Comment:
none of the tasks test pass

Reply:
weirdly enough, they pass for me locally